pkg/innerring/processors/container/common.go

```diff
@@ -66,16 +67,12 @@ func (cp *Processor) verifySignature(v signatureVerificationData) error {
 			return fmt.Errorf("decode session token: %w", err)
 		}
 
-		if !tok.VerifySignature() {
+		tokSigner, err := icrypto.VerifyTokenSignature(&tok)
+		if err != nil {
 			return errors.New("invalid session token signature")
 		}
 
-		var signerPub neofsecdsa.PublicKeyRFC6979
-		if err = signerPub.Decode(tok.IssuerPublicKeyBytes()); err != nil {
-			return fmt.Errorf("invalid issuer public key: %w", err)
-		}
-
-		if user.NewFromECDSAPublicKey(ecdsa.PublicKey(signerPub)) != v.ownerContainer {
+		if tokSigner != v.ownerContainer {
```

Comment:
Technically, this should be a token issuer, not signer. And then token verification routine needs to check that issuer is the same as signer. But it works at this stage.

Reply:
done